Library/Formula/gpg-agent.rb

```diff
@@ -33,9 +33,46 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+      To replace ssh-agent with gpg-agent, you need to:
+         * ensure that "enable-ssh-support" is in ~/.gnupg/gpg-agent.conf
+         * ensure that "pinentry" in ~/.gnupg/gpg-agent.conf has a graphical PIN entry
+           such as pinentry-mac (needed for GUI programs)
+         * disable the system ssh-agent with
+             launchctl unload -w /System/Library/LaunchAgents/org.openssh.ssh-agent.plist
+         * have launchd start gpg-agent at login
+    EOS
+  end
```

Comment:
Ah, I see, my bad, I thought you meant the plist from an existing formula. Too early in the morning! I think in that case I'd rather leave it as an exercise for the reader; we prefer not to encourage people to mess with system services like this.